common/src/main/java/tachyon/exception/NotFoundException.java

```diff
@@ -18,7 +18,9 @@
 /**
  * Exception used when some requested entity (e.g., block, file, directory or lock) was not found.
  */
-public class NotFoundException extends Exception {
+public final class NotFoundException extends AbstractTachyonException {
+  private static final long serialVersionUID = 6901397458237323517L;
```

Comment:
Same naming convention issue here, (and applies to other static final members in this PR)